control-plane/pkg/reconciler/source/auth_provider.go

```diff
@@ -28,23 +28,20 @@ func (ks *SecretLocator) SecretName() (string, bool) {
 	if !hasAuthConfig(ks.KafkaSource) {
 		return "", false
 	}
-	return getName(ks.KafkaSource), hasAuthConfig(ks.KafkaSource)
+	// KafkaSource uses a special secret provider, so there is no need to return a name here.
```

Comment:
special, as in? mind linking to details?

Reply:
Let me know if the new commit solves your comment